ml/dgl_utils.py

```diff
@@ -0,0 +1,253 @@
+# classes for converting a dataframe or Graphistry Plottable into a DGL
+import pandas as pd
+import dgl
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import umap
+import numpy as np
+
+from ml.utils import pandas2dgl
+from ml.utils import process_dirty_dataframes
+from ml import constants as config
+
+import logging
+
+logger = logging.getLogger(__name__)
+FORMAT = "[%(filename)s:%(lineno)s - %(funcName)20s() ] %(message)s"
+logging.basicConfig(format=FORMAT)
+logger.setLevel(logging.DEBUG)
+
+
+# ndata_fields = ['feature', 'label', 'train_mask', 'test_mask', 'validation_mask']
+# edata_fields = ['feature', 'relationship', 'train_mask', 'test_mask', 'validation_mask']
+
+
+def get_vectorizer(name):
```

Comment:
we try to do py3 mypy-style type annotations ("`def get_vectorizer(name: str) -> ???`") , esp, for anything public (and full doc strings then)